manifest-schema.graphql

```diff
@@ -27,14 +27,25 @@ type EthereumContractMapping {
   file: File!
   entities: [String!]!
   abis: [EthereumContractAbi!]!
-  eventHandlers: [EthereumContractEventHandlers]!
+  blockHandlers: EthereumBlockHandler
+  transactionHandlers: [EthereumTransactionHandlers!]
+  eventHandlers: [EthereumContractEventHandlers!]
 }
 
 type EthereumContractAbi {
   name: String!
   file: File!
 }
 
+type EthereumBlockHandler {
+  handler: String!
+}
+
+type EthereumTransactionHandlers {
```

Comment:
You mentioned in Slack this should be singular, not plural. Should we also make the change for `EthereumContractEventHandlers`?

Reply:
Yeah, that makes sense!